src/webhooks.ts

```diff
@@ -75,4 +75,38 @@ export class WebhooksApi {
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
 
+  /*
+   * A function to update an existing Marqeta Webhook based upon
+   * an updated webhook as the input argument. The required Marqeta
+   * parameters to update a webhook are: url, events, basicAuthName,
+   * basicAuthPassword, name, and. token.
+   */
+  async update (hook: WebHook): Promise<{
```

Comment:
You have `WebHook` and `WebhookList`... and the question is: Why not `Webhook` or `WebHookList`? What's the motivation for the different case on the `H`?